src/components/GlobalCheckClaimStatus/index.tsx

```diff
@@ -29,17 +31,35 @@ const GlobalCheckClaimStatus: React.FC<GlobalCheckClaimStatusProps> = (props) =>
  */
 const GlobalCheckClaim: React.FC<GlobalCheckClaimStatusProps> = ({ excludeLocations }) => {
   const hasDisplayedModal = useRef(false)
-  const [canClaimAnniversaryPoints, setCanClaimAnniversaryPoints] = useState(false)
+  const [cid, setCid] = useState(null)
+  const [canClaimNFT, setCanClaimNFT] = useState(false)
   const { account } = useWeb3React()
   const { pathname } = useRouter()
-  const [onPresentAnniversaryModal] = useModal(<AnniversaryAchievementModal />)
+  const [onPresentModal] = useModal(<GalaxyNFTClaimModal cid={cid} />, false, true, 'galaxyNFTClaimModal')
 
   // Check claim status
   useEffect(() => {
     const fetchClaimAnniversaryStatus = async () => {
-      const { canClaim } = getAnniversaryAchievementContract()
-      const canClaimAnniversary = await canClaim(account)
-      setCanClaimAnniversaryPoints(canClaimAnniversary)
+      try {
+        const { campaign } = await request(
+          GALAXY_NFT_CLAIMING_API,
+          gql`
+            query checkEligabilityForGalaxyNFT($campaignId: ID!, $address: String!) {
```

Comment:
That typo is on their end right?

Reply:
nah, those are local query names, my typo - fixed, thanks

Reply:
ah yes you are right 👍